bindings/resvg-skia-safe/src/lib.rs

```diff
@@ -0,0 +1,526 @@
+use std::ops::{Deref, DerefMut};
```

Comment:
Can we rename `resvg-skia-safe/src/lib.rs` to `resvg-skia/src/lib.rs` (and remove the superfluous files), so that it's easier to see that the interfaces are the same, all are covered, and identify new ones/changes? That should mean that we can remove the feature flag as well.

The disadvantage of this is that we won't be able to get it merged upstream, but I don't think that's a problem.

Reply:
That was actually my initial approach, but I changed it in the end to keep the original bindings. The logic behind that decision was that it would be easy to see while debugging issues whether they are caused by a general problem with resvg or by just our bindings - you would just render an SVG with their bindings and see if the problem persists. Well, "easy" is a strong word considering that I wasn't even able to build it with their bindings. In any case, that is also achievable by having two checkouts of resvg.

Reply:
done